cmd/k8s-operator/operator.go

```diff
@@ -127,9 +127,14 @@ func initTSNet(zlog *zap.SugaredLogger) (*tsnet.Server, *tailscale.Client) {
 	tsClient := tailscale.NewClient("-", nil)
 	tsClient.HTTPClient = credentials.Client(context.Background())
 
+	configDir, err := os.MkdirTemp(os.TempDir(), "tsnet-config")
+	if err != nil {
+		startlog.Fatalf("error creating config directory for tsnet: %v", err)
+	}
 	s := &tsnet.Server{
 		Hostname: hostname,
 		Logf:     zlog.Named("tailscaled").Debugf,
+		Dir:      configDir,
```

Comment:
This does not contain tailscaled state (on kube that's written to a Secret). It only contains a `tailscaled.log.conf` file with log IDs